.circleci/config.yml

```diff
@@ -17,32 +17,45 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-          - v3-dependencies-{{ checksum "Gemfile.lock" }}
+          - v4-dependencies-{{ checksum "Gemfile.lock" }}
           # fallback to using the latest cache if no exact match is found
-          - v3-dependencies-
+          - v4-dependencies-
```

Comment:
that's going to be a bit of an issue, you'd have to manually change - can you keep at least the {{ checksum "Gemfile.lock" }} so if a new gem change is needed it'll just work and would only be manual change v4->v5 if changing markdown etc... there is also probably a "proper" lock file of sort for npm too